rest/client/mock.go

```diff
@@ -55,7 +55,8 @@ type Mock struct {
 	logMessages map[string][]apimodels.LogMessage
 	PatchFiles  map[string]string
 	keyVal      map[string]*serviceModel.KeyVal
-	mu          sync.Mutex
+
+	Mu sync.Mutex
```

Comment:
I wouldn't export the mutex like this (you can just anonymously compose the mutex, if you must)